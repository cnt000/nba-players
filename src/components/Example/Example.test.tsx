import React from 'react'
import { render } from '../../test/testUtils'
import Example from './Example'

const labels = {
  testLabel: 'hello king'
}

const Component = <Example id={'test'} labels={labels} />

describe('Example', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(Component, {})
    expect(asFragment()).toMatchSnapshot()
  })

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(Component, {})
  //   window.alert = jest.fn()
  //   fireEvent.click(getByText('Test Button'))
  //   expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
  // })
})
