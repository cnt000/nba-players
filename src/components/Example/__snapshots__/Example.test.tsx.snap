// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdDrOH hqPdxJ"
  >
    hello Vert King test: {"testLabel":"hello king"}
  </h1>
  <button>
    Test Button
  </button>
</DocumentFragment>
`;
